### PR TITLE
Person Card (New Individual Mom Box)

### PR DESCRIPTION
**Pull request by DallasLind**
Created a new branch due to branching issues with another component I was constructing. Currently working on CSS and will convene with Jenna about the profile pic.

### CODE REVIEW

File: client/src/components/PersonCard/personcard.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const PersonCard = (props) => {
+
+  // Default Class to apply to Component
+  let classList = `PersonCard`;
+
+
+  return(
+    <div className={classList}>
+      <img className="PersonCard__Img" img src={mom} />
+      <p className="PersonCard__MomName">{props.PersonCard__MomName}</p>
```
Comment (dcartist) [inline client/src/components/PersonCard/personcard.js]: Should the mom name be more semantic?

File: client/src/components/PersonCard/personcard.stories.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PersonCard from './personcard';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('PersonCard', module)
+  .add("Default", () => <PersonCard
+  PersonCard__MomName="Lindsay L."
```
Comment (dcartist) [inline client/src/components/PersonCard/personcard.stories.js]: Should the mom name be more semantic?

File: client/src/components/PersonCard/personcard.js (hunk 1 above)
Comment (DallasLind) [reply client/src/components/PersonCard/personcard.js]: Changed to ProfileName

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#392C04",
+        "titleBar.activeBackground": "#503D06",
+        "titleBar.activeForeground": "#FEFAEF"
+    }
+}
```
Comment (ZakkMan) [inline .vscode/settings.json]: Can you get ride of this file?

File: client/src/App.css [hunk 4]
```diff
@@ -1,5 +1,4 @@
 @import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,700&display=swap');
```
Comment (ZakkMan) [inline client/src/App.css]: Can you undelete this?

File: client/src/components/PersonCard/personcard.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
+.PersonCard {
+width: 229px;
+height: 289px;
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: Can you use responsive height and widths rather than explicit pixel values?

Comment (ZakkMan) [reply client/src/components/PersonCard/personcard.css]: Also your CSS properties should be indented!

File: client/src/components/PersonCard/personcard.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+border: 1px solid var(--yellow);
+background: var(--white);
+position: absolute;
+left: 0%;
+right: 0%;
+top: 0%;
+bottom: 0%;
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: Why is this absolutely positioned?

File: client/src/components/PersonCard/personcard.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.PersonCard__Img {
+position: absolute;
+width: 150px;
+height: 150px;
+left: 35px;
+top: 12px;
+}
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: Could you use flexbox to layout the items inside the `PersonCard`? Also, the modifier or element should be lower case (i.e. `.PersonCard__image`)

File: client/src/components/PersonCard/personcard.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+.PersonCard__Location {
+font-family: var(--default-font);
+font-weight: var(--title-weight);
+position: absolute;
+left: 17.03%;
+right: 17.03%;
+top: 63.97%;
+bottom: 0%;
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: We want to minimize the number of things that are absolutely positioned!

File: client/src/components/PersonCard/personcard.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+width: 150px;
+height: 150px;
+left: 35px;
+top: 12px;
+}
+
+.PersonCard__ProfileName {
+font-family: var(--default-font);
+font-weight: var(--paragraph-weight);
+font-size: 30px;
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: If you're using `p` and `h` tags, then you shouldn't have to set the font-size on everything

File: client/src/components/PersonCard/personcard.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import "./personcard.css";
+import mom from "../Assets/Images/Mom.png"
+
+// Function based React Component
+const PersonCard = (props) => {
+
+  // Default Class to apply to Component
+  let classList = `PersonCard`;
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.js]: Can you put this class in the `className` attribute, rather than create an extra variable for it?

File: client/src/components/PersonCard/personcard.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+      <img className="PersonCard__Img" img src={mom} />
+      <p className="PersonCard__ProfileName">{props.PersonCard__ProfileName}</p>
+      <div className="PersonCard__MomBoxInfo">
+        <p className="PersonCard__Location">{props.PersonCard__Location}</p>
+        <p className="PersonCard__Interests">
+          {props.PersonCard__Interests}
+        </p>
+      </div>
+    </div>
+  );
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.js]: This looks great! Very clean!

File: client/src/components/PersonCard/personcard.stories.js [hunk 12]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PersonCard from './personcard';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('PersonCard', module)
+  .add("Default", () => <PersonCard
+  PersonCard__ProfileName="Lindsay L." 
+  PersonCard__Location="Washington D.C." 
+  PersonCard__Interests="#budgeting #technology #outdoors #traveling"
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.stories.js]: Can we make these props names shorter? i.e. `name` and `location`? I think `interests` should maybe be an array

Comment (ZakkMan) [reply client/src/components/PersonCard/personcard.stories.js]: Also, one thing to think about is we don't know if those hashtags are supposed to be clickable (eventually) or not!

File: client/src/components/PersonCard/personcard.css [hunk 13]
```diff
@@ -0,0 +1,52 @@
+.PersonCard {
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.css]: Can you rename this file back to `PersonCard`?

File: client/src/components/PersonCard/personcard.js [hunk 14]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.js]: Can you rename this one back to `PersonCard.js` too?

File: client/src/components/PersonCard/personcard.js [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+import "./personcard.css";
+import mom from "../Assets/Images/Mom.png"
+
+// Function based React Component
+const PersonCard = props => {
+  return(
+    <div className="PersonCard">
+      <img className="PersonCard__image" img src={mom} />
+      <div className="PersonCard__name">{props.PersonCard__name}</div>
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.js]: Can you fix these `prop` names?

File: client/src/components/PersonCard/personcard.stories.js [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+storiesOf('PersonCard', module)
+  .add("Default", () => <PersonCard
+  PersonCard__name="Lindsay L." 
+  PersonCard__location="Washington D.C." 
+  PersonCard__interests={[ 
+  { url: "#", PersonCard__interests: "veganism" },
+  { url: "#", PersonCard__interests: "budgeting" },
+  { url: "#", PersonCard__interests: "travel" },
+  { url: "#", PersonCard__interests: "outdoors"}
+]}
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.stories.js]: Can you make these prop names shorter? i.e. `name`, `location`, `interests`

File: client/src/components/PersonCard/personcard.js [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+      <div className="PersonCard__MomBoxInfo">
+        <div className="PersonCard__location">{props.PersonCard__location}</div>
+        <div className="PersonCard__interests">
+          {props.PersonCard__interests.map(PersonCard__interests => (
+              <a className="PersonCard__interests-links" href={PersonCard__interests.url}>
+                #{PersonCard__interests.PersonCard__interests}
+              </a>
+            ))}
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.js]: Can you use a `Link` component?

File: client/src/components/PersonCard/personcard.test.js [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
 	// we will write one individual test
   it('should render as expected', () => {
     // Shallow rendering renders a component without rendering any of its children
-    const component = shallow(<Logo />)
+    const component = shallow(<PersonCard />)
     // We create an assertion within the test that checks if our component renders our name prop
   // expect(component.contains('Your name')).toBe(true)
   })
-})
+})
```
Comment (ZakkMan) [inline client/src/components/PersonCard/personcard.test.js]: What happened here? Did you move the `Logo.test.js` file?

Comment (DallasLind) [reply client/src/components/PersonCard/personcard.test.js]: I'm actually not sure, I don't remember what happened with this one. Is there something I should do to rectify or change the test.js?

Comment (ZakkMan) [reply client/src/components/PersonCard/personcard.test.js]: You can revert the single file!

File: client/src/components/PersonCard/PersonCard.js [hunk 19]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import "./PersonCard.css";
+import mom from "../Assets/Images/Mom.png"
+
+// Function based React Component
+const PersonCard = props => {
+  return(
+    <div className="PersonCard">
+      <img className="PersonCard__image" img src={mom} />
```
Comment (ZakkMan) [inline client/src/components/PersonCard/PersonCard.js]: Sorry I only just caught this! We'll want to pass the image down as a prop!

File: client/src/components/Logo/Logo.test.js [hunk 20]
```diff
@@ -1,16 +0,0 @@
-import React from 'react'
-import { shallow, configure } from 'enzyme'
-import Logo from './Logo.js'
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
-
-// We will describe a block of tests
-describe('Logo component', () => {
```
Comment (ZakkMan) [inline client/src/components/Logo/Logo.test.js]: Why was this file deleted?

Comment (DallasLind) [reply client/src/components/Logo/Logo.test.js]: I super don't remember/have no idea what happened with it

File: client/src/components/PersonCard/PersonCard.stories.js [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+storiesOf('PersonCard', module)
+.add("Default", () => <PersonCard
+  PersonCard__image type="large" image={Image} 
+  PersonCard__name="Lindsay L." 
+  PersonCard__location="Washington D.C." 
+  PersonCard__interests={[ 
+  { url: "#", interests: "veganism" },
+  { url: "#", interests: "budgeting" },
+  { url: "#", interests: "travel" },
+  { url: "#", interests: "outdoors"}
```
Comment (ZakkMan) [inline client/src/components/PersonCard/PersonCard.stories.js]: Didn't you fix these prop names in a previous PR? These should just be `image`, `name`, `location`, etc - not `PersonCard__image`

File: client/src/components/PersonCard/PersonCard.js [hunk 22]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import "./PersonCard.css";
+import ProfilePicture from "../ProfilePicture/ProfilePicture.js";
```
Comment (ZakkMan) [inline client/src/components/PersonCard/PersonCard.js]: The image should be passed down as a prop, not importing inside the component
